### DISCUSSION TITLE
How to download googleapis release version

### DISCUSSION THREAD

**Discussion by aks-zero** [googleapis/googleapis#752] [Q&A]
In our application we use `googleapis` for a gRPC based voice service. We compile only some of the common code e.g. `google/api, google/rpc, google/type` etc. along with the custom proto file to build a C++ library to be used by the application.

It's technically working. But the issue we are having is that the repo doesn't have release versions or tags. We are forced to download code from master branch. The automated build always gets the latest from master which is kind of risky. We want to ensure that the build always downloads the same code until changed e.g. a tagged version instead of live branch.

Why this repos doesn't have tags like other repos? Currently we clone the master and then checkout a specific commit by commit hash, which is an ugly hack. Can someone please review and suggest a better way?

**Comment by unmultimedio** [comment]
Similar ask than https://github.com/googleapis/googleapis/discussions/856, having stable semver releases to pin usages of this repo is highly desired instead of tracking the `master` branch. Any updates on this?
